codice/alliance
# rebrand for transfer to Codice

#### What does this PR do?
Rebrand, repackage, refactor for transfer of alliance source code to Codice
#### Who is reviewing it?
@bdeining @troymohl @kcwire @jlcsmith 
#### How should this be tested?
Full build against a clean repo
Unzip distro, start Alliance, and verify all apps start succesfully.
#### Any background context you want to provide?

#### What are the relevant tickets?

#### Screenshots (if appropriate)
#### Checklist:
